### DISCUSSION TITLE
Hello dear Dr

### DISCUSSION THREAD

**Discussion by WTK1998** [PMEAL/OpenPNM#2692] [Q&A]
Hello dear Dr.
how can I calculate drainage two phase relative permeability with mixed invasion percolation.(with trapping) (openpnm 2.8.2)
I want to compare that with invasion percolation rel perm.
Thanks.

**Comment by jgostick** [comment]
We used to have detailed example notebooks on that.  Through the magic of git versioning, you can still find them here: https://github.com/PMEAL/OpenPNM/tree/v2-latest/examples/simulations/percolation

**Reply by WTK1998** [reply]
Thanks.
I have problem in calculating Mixed Invasion relative permeability when snap off and trapping are True.
